### PR TITLE
Update URLs in libraries.js

### PR DESCRIPTION
**Pull request by eolognt**
Hi, I am using this great extension, but found that some URLs to the libraries home pages were dead. So I took the liberty of going through all URLs and updated the ones that were dead. There was also a link to a GitHub repo were the library had an own webpage, which URL I changed to.

### CODE REVIEW

File: library/libraries.js [hunk 1]
```diff
@@ -660,7 +660,7 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
 
     'ControlJS': {
         icon: 'icon_48',
-        url: 'http://stevesouders.com/controljs/',
+        url: 'http://controlsjs.com/',
```
Comment (johnmichel) [inline library/libraries.js]: This is the wrong URL for this library.  The original is still online and refers to the correct library detection test.

If you'd like, you can add an entry to include the library used at [controlsjs.com](http://controlsjs.com) and send an additional PR.

File: library/libraries.js [hunk 2]
```diff
@@ -738,7 +738,7 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
 
     'Move': {
        icon: 'move',
-       url: 'http://movelang.org/',
+       url: 'https://visionmedia.github.io/move.js/',
```
Comment (johnmichel) [inline library/libraries.js]: This is the wrong URL for this library.  Unfortunately, the original domain no longer contains information about it.  See this for more details: [Move — a programming language](https://rsms.me/move-programming-language)

If you'd like, you can revise this entry to use new tests for the current Move library URL.

**Comment by eolognt** [comment]
@johnmichel Hi, I reverted the first commented diff and used this url: https://github.com/rsms/move for the second diff.

**Comment by johnmichel** [comment]
Thanks!

**Pull request merged**